ai21/errors.py

```diff
@@ -95,8 +95,10 @@ def __init__(self, key: str):
 
 
 class StreamingDecodeError(AI21Error):
-    def __init__(self, chunk: str):
+    def __init__(self, chunk: str, error_message: Optional[str] = None):
         message = f"Failed to decode chunk: {chunk} in stream. Please check the stream format"
+        if error_message:
+            message += f". Error: {error_message}"
```

Comment:
Weird formatting. Add the period - . at the end of the actual `message` and add whatever message you want after